x-pack/plugins/ml/public/components/rule_editor/scope_section.js

```diff
@@ -23,18 +23,34 @@ import {
 import { ScopeExpression } from './scope_expression';
 import { checkPermission } from '../../privilege/check_privilege';
 import { getScopeFieldDefaults } from './utils';
+import { FormattedMessage } from '@kbn/i18n/react';
 
 
 function NoFilterListsCallOut() {
   return (
     <EuiCallOut
-      title="No filter lists configured"
+      title={<FormattedMessage
+        id="xpack.ml.ruleEditor.scopeSection.noFilterListsConfiguredTitle"
+        defaultMessage="No filter lists configured"
+      />}
       iconType="gear"
     >
       <p>
-        To configure scope, you must first use the&nbsp;
-        <EuiLink href="#/settings/filter_lists">Filter Lists</EuiLink> settings page
-        to create the list of values you want to include or exclude in the rule.
+        <FormattedMessage
+          id="xpack.ml.ruleEditor.scopeSection.howToConfigureScopeDescription"
```

Comment:
Change id to use `createFilterListsDescription` rather than `howToConfigureScopeDescription`